document_knowledge/models/res_config.py

```diff
@@ -9,25 +9,17 @@ class DocumentKnowledgeConfigSettings(models.TransientModel):
 
     module_attachment_indexation = fields.Boolean(
         "Attachments List and Document Indexation",
-        help="Document indexation, full text search of attachements.\n"
```

Comment:
I prefer that you remove all the options, as this is not valid if the rest of the modules aren't there.

Reply:
@pedrobaeza Thanks for your time!!
Can you explain more so I can better understand what you want.

Reply:
I mean that having these `module_*` fields is for installing modules that are still not migrated, but even if migrated, you may not have those modules present on your Odoo instance, so clicking on the checks provoke confusion and misleading in users, so I propose to remove these fields from the configuration.